app/questionnaire/rules.py

```diff
@@ -69,6 +69,20 @@ def evaluate_repeat(repeat_rule, answer_store):
         return repeat_function(filtered)
 
 
+def evaluate_skip_condition(skip_condition, answer_store):
```

Comment:
Pretty sure there is already at least one if not more of these identical blocks of code about parsing/evaluating when rules, can it be consolidated?